lib/constant.go

```diff
@@ -1,9 +1,37 @@
 package sebak
 
-import "boscoin.io/sebak/lib/common"
+import (
+	"time"
+
+	"boscoin.io/sebak/lib/common"
+)
 
 const (
 	// BaseFee is the default transaction fee, if fee is lower than BaseFee, the
 	// transaction will fail validation.
 	BaseFee sebakcommon.Amount = 10000
 )
+
+var (
+	// TimeoutProposeNewBallot works when the consensus process is finished for
+	// a proposed `Ballot`, proposer will wait for `TimeoutProposeNewBallot` and
+	// then it will propose new `Ballot`.
+	TimeoutProposeNewBallot time.Duration = time.Second * 2
+
+	// TimeoutProposeNewBallotFull is almost same with `TimeoutProposeNewBallot`,
+	// but if `Transaction`s in `TransactionPool` is over
+	// `MaxTransactionsInBallot`, proposer will wait for
+	// `TimeoutProposeNewBallotFull`.
+	TimeoutProposeNewBallotFull time.Duration = time.Second * 1
+
+	// TimeoutExpireRound works if running `Round` is expired for consensus.
+	TimeoutExpireRound time.Duration = time.Second * 10
+
+	// MaxTransactionsInBallot limits the maximum number of `Transaction`s in
+	// one proposed `Ballot`.
+	MaxTransactionsInBallot int = 1000
+
+	// MaxOperationsInTransaction limits the maximum number of `Operation`s in
+	// one `Transaction`.
+	MaxOperationsInTransaction int = 200
+)
```

Comment:
Those refactoring, while welcome, would have been better in a separate commit.
The point is that if we want to track down a problem (via `git bisect`) or fix one (`git revert`), we don't want to have to deal with unrelated / independent changes which are worthwhile.

Reply:
@Geod24 OK, good point. We can handle `MaxOperationsInTransaction` in new issue.